src/Microsoft.VisualStudio.Editor.Razor/RazorDirectiveCompletionProvider.cs

```diff
@@ -86,6 +103,13 @@ public override Task ProvideCompletionsAsync(CompletionContext context)
                 return Task.CompletedTask;
             }
 
+            if (!_textBufferProvider.TryGetFromDocument(context.Document, out var textBuffer) ||
+                !_asyncCompletionBroker.IsCompletionSupported(textBuffer.ContentType))
```

Comment:
@AmadeusW Here's my take on turning off our current completion system based off of your feature flag work. Can't verify end-to-end yet but highlighting for now to get your thoughts.

Reply:
this will work 👍👍